packages/ember/index.js

```diff
@@ -651,8 +651,36 @@ if (ENV.EXTEND_PROTOTYPES.String) {
     return htmlSafe(this);
   };
 }
-Ember.String.htmlSafe = htmlSafe;
-Ember.String.isHTMLSafe = isHTMLSafe;
+const deprecateImportFromString = function (
+  name,
+  message = `Importing ${name} from '@ember/string' is deprecated. Please import ${name} from '@ember/template' instead.`
+) {
+  deprecate(message, false, {
+    id: 'ember-string.htmlsafe-ishtmlsafe',
+    for: 'ember-source',
+    since: {
+      available: '3.25',
+    },
```

Comment:
Can you set `enabled` instead of `available`?

```suggestion
    since: {
      enabled: '3.25',
    },
```


Until the deprecation stages RFC is fully rolled out, we are just using `enabled` for deprecations (it means that folks get the deprecation right away and don't have to opt in).